spec_diffs/2021-2023

```diff
@@ -40,6 +40,7 @@
 -     "_enum": Object {
 -       "CINS": "[u8; 9]",
 -       "CUSIP": "[u8; 9]",
+-       "FIGI": "[u8; 12]",
```

Comment:
Isn't this an old file. I wasn't expecting this one to get changed

Reply:
Diff files are a bit missed up, I have fixed the sequence and generated fresh diffs again